services/tenant-ui/frontend/src/helpers/index.ts

```diff
@@ -0,0 +1,33 @@
+import { format, parseJSON } from 'date-fns'
```

Comment:
Oh.... haha. I see you used this thing for date formatting. Still... Must be much smaller then other tools.

Reply:
> Oh.... haha. I see you used this thing for date formatting. Still... Must be much smaller then other tools.

there is [Intl.DateTimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat) (native) that maybe we can use? maybe pin that for later, could take a while to figure out.

Reply:
but that leads to a better question... should the traction api be returning a string or a numeric value for timestamps.  probably... both? at least with numeric it's much easier for callers to format themselves.

Reply:
Yeah Moment is deprecated now (https://momentjs.com/docs/#/-project-status/) and most of the Googling about what to do instead suggests date-fns.

Wrapping it in that helper can let us swap it out for anything else in the future too